src/components/App/index.js

```diff
@@ -93,6 +94,18 @@ class ModalApp extends Component {
         this.props.actions.setUrls(jwtUrls)
       }
     }
+
+    if (options.enterpriseFeatures && token && token !== prevToken) {
+      const validEnterpriseFeatures = getEnterpriseFeaturesFromJWT(token)
```

Comment:
Should we return the `onError` callback when expected enterpriseFeatures properties cannot be retrieved from the JWT?

Reply:
Thanks, I forgot about that.